test/Psr/CacheItemPool/CacheItemTest.php

```diff
@@ -106,4 +110,32 @@ public function testExpiresAfterInvalidThrowsException(): void
         $item = new CacheItem('key', 'value', true);
         $item->expiresAfter([]);
     }
+
+    public function testExpiresAfterStartsExpiringAfterMethodCall(): void
+    {
+        $item = new CacheItem('key', 'value', false);
+        $item = $item->expiresAfter(10);
+        sleep(1);
```

Comment:
Shouldn't be needed, with the new clock? Or are you explicitly testing the system clock default behavior here?